ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-release-4.8.yaml

```diff
@@ -0,0 +1,182 @@
+base_images:
+  must-gather:
+    name: "4.8"
+    namespace: ocp
+    tag: must-gather
+  operator-registry:
+    name: "4.8"
+    namespace: ocp
+    tag: operator-registry
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.15-openshift-4.7
```

Comment:
does a 4.8 tag not exist (yet?)

Reply:
Oh, nope, missed that! Changed it to the same one as used in master.

Reply:
👍